perllib/FixMyStreet/Cobrand/Bexley.pm

```diff
@@ -99,6 +99,27 @@ ID.
 
 =cut
 
+=item * Customised pin colours
+
+Bexley has supplied their own colours for pins #4521
+
+=cut
+
+sub path_to_pin_icons {
+    return '/cobrands/bexley/images/';
+}
+
+sub pin_colour {
+    my ( $self, $p ) = @_;
+
+    return 'aqua' if $p->state eq 'investigating';
+    return 'orange' if $p->state eq 'action_scheduled';
+    return 'green' if $p->is_fixed;
+    return 'spring' if $p->is_closed;
+    return 'grape' if $p->state eq 'not_council_responsibility';
```

Comment:
This line will never be reached because it'll match on the is_closed line above.